<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Jours</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_31">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>15:00 - 15:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>00:00 - 00:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>03:00 - 03:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>08:30 - 09:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_48">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>500</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>23:30 - 24:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_40">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>340</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>19:30 - 20:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_38">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>18:30 - 19:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>05:30 - 06:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_46">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>460</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>22:30 - 23:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>07:30 - 08: 00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_32">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>15:30 - 16:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>04:30 - 05:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_26">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>12:30 - 13:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>06:00 - 06:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>00:30 - 01:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>08:00 - 08:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>04:00 - 04:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_37">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>18:00 - 18:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>10:00 - 10:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>06:30 - 07:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_28">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>13:30 - 14:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>09:00 - 09:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>03:30 - 04:00</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Lundi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mardi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mercredi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jeudi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vendredi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Samedi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dimanche</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="checkBox_34">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>16:30 - 17:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_27">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>13:00 - 13:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_44">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>420</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>21:30 - 22:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_42">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>380</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>20:30 - 21:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_35">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>240</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>17:00 - 17:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>09:30 - 10:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_39">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>320</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>19:00 - 19:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>01:30 - 02:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>02:00 - 02:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_36">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>17:30 - 18:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_25">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>12:00 - 12:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>07:00 - 07:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>02:30 - 03:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_41">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>360</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>20:00 - 20:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>10:30 - 11:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>01:00 - 01:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>05:00 - 05:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_29">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>14:00 -14:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_45">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>440</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>22:00 - 22:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_47">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>480</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>23:00 - 23:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>11:00 - 11:30</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>530</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Enregistrer</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_43">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>21:00 - 21:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_33">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>16:00 - 16:30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_30">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>14:30 - 15:00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>11:30 - 12:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_validation">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>560</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Les données ont bien été enregistrées !</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Humidité</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Taux humidité :</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_tauxHumidite">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_end">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Enregistrer et terminer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_traitement">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Le traitement est terminé !</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
